boards/imixv2/Makefile-app

```diff
@@ -0,0 +1,21 @@
+# Makefile for loading applications for the Imix platform
+
+$(call check_defined, TOCK_USERLAND_BASE_DIR)
+$(call check_defined, BUILDDIR)
+$(call check_defined, PACKAGE_NAME)
+
+TOCK_ARCH = cortex-m4
+BOARD_BUILDDIR = $(BUILDDIR)/$(TOCK_ARCH)
+
+APP_FLASH = $(TOCK_USERLAND_BASE_DIR)/tools/flash/imix.sh
```

Comment:
I assume this can be updated to use Tockloader as well? Do we need this Makefile-app at all, or can the default one be used?